themes/bootstrap3/scss/components/autocomplete.scss

```diff
@@ -1,4 +1,5 @@
-/* https://github.com/vufind-org/autocomplete.js 1.0b */
+/* https://github.com/vufind-org/autocomplete.js (v2.1.3) */
```

Comment:
Should this file move to vendor for consistency with the LESS version? Otherwise, I don't think the imports are going to work right.